src/components/PostEditor.js

```diff
@@ -146,25 +146,20 @@ export class PostEditor extends React.PureComponent {
   }
 
   validate () {
-    let { postEdit } = this.props
-    const { title, subeditor } = this.refs
+    let { parentPost, postEdit } = this.props
+    const { title } = this.refs
 
     if (!postEdit.name) {
       window.alert('The title of a post cannot be blank.')
       title.focus()
       return Promise.resolve(false)
     }
 
-    if (isEmpty(postEdit.community_ids)) {
+    if (!parentPost && isEmpty(postEdit.community_ids)) {
       window.alert('Please pick at least one community.')
       return Promise.resolve(false)
     }
 
-    if (subeditor) {
```

Comment:
I can see why you ended up dropping this whole branch of code, since the only thing we were validating across both events and posts was the tag... I just hope we don't end up needing it again for other fields.